src/menus/icon-menu.jsx

```diff
@@ -180,8 +180,11 @@ const IconMenu = React.createClass({
       muiTheme: nextContext.muiTheme || this.state.muiTheme,
     });
 
-    if (nextProps.open === true || nextProps.open === false) {
-      this.setState({open: nextProps.open});
+    if (nextProps.open !== null) {
+      this.setState({
+        open: nextProps.open,
+        anchorEl: ReactDOM.findDOMNode(this.refs.iconMenuContainer),
```

Comment:
`findDOMNode` is not needed as the target is a dom element. Ideally, it would be even better if you could use callback refs.